### ISSUE TITLE
Failing nightly builds due to Omegah

### ISSUE THREAD

**Issue by ikalash** [sandialabs/Albany#1043]
There are test failures in the nightlies involving Omegah: https://sems-cdash-son.sandia.gov/cdash/viewBuildError.php?buildid=68010 

E.g., 

```
[Albany/tests/unit/disc/omegah/box_mesh.cpp:51](https://blob/master/Albany/tests/unit/disc/omegah/box_mesh.cpp#L51):46: error: 'class Teuchos::RCP<Omega_h::Mesh>' has no member named 'owned'
```
@bartgol I think this is due to your recently merged PR.  Can you please fix?  Is it b/c I am not using Omegah in the nightlies and there is a missing ifdef?

**Comment by bartgol** [comment]
Yes, my bad. I did not build the unit tests to verify everything was ok. f89da1a5278f9cd should fix it.
